examples/notebooks/tsne_mnist.ipynb

```diff
@@ -0,0 +1,328 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "for emb, title in zip([embedding_intelex, embedding_original], \n",
+    "                      ['Intel® Extension for scikit-learn', 'Original scikit-learn']):\n",
+    "    plt.scatter(emb[:, 0], emb[:, 1], c=colors)\n",
+    "    plt.title(title)\n",
+    "    plt.xlabel('x')\n",
+    "    plt.ylabel('y')\n",
+    "    plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "ffd79e96",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<h2>With scikit-learn-intelex patching you can:</h2><ul><li>Use your Scikit-learn code for training and prediction with minimal changes (a couple of lines of code);</li><li>Fast execution training and prediction of Scikit-learn models;</li><li>Get the similar quality</li><li>Get speedup in <strong>3.4</strong> times.</li></ul>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "HTML(f'<h2>With scikit-learn-intelex patching you can:</h2>'\n",
+    "     f\"<ul>\"\n",
+    "     f\"<li>Use your Scikit-learn code for training and prediction with minimal changes (a couple of lines of code);</li>\"\n",
+    "     f\"<li>Fast execution training and prediction of Scikit-learn models;</li>\"\n",
+    "     f\"<li>Get the similar quality</li>\"\n",
+    "     f\"<li>Get speedup in <strong>{(time_original/time_opt):.1f}</strong> times.</li>\"\n",
+    "     f\"</ul>\")"
```

Comment:
![image](https://user-images.githubusercontent.com/24354279/208718335-b34dfe28-034b-43d9-a3be-a9798dd67ce0.png)
Github file preview doesn't look nice for this cell

Reply:
Removed.